docs/pywbemcli/generaloptions.rst

```diff
@@ -725,6 +750,12 @@ any subsequent pywbemcli commands:
     . . . <list of namespaces for the defined server>
 
 
+.. index::
+    pair: pull operations; general options
+        single: --use-pull
+        single: --pull-max-count
+        single: traditional operations
```

Comment:
not sure if indentation matters, but maybe align

Reply:
DONE